packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -326,13 +326,11 @@ export class CrossChainMessenger {
    * Transforms a legacy message into its corresponding Bedrock representation.
    *
    * @param message Legacy message to transform.
+   * @param messageIndex The index of the message, if multiple exist from multicall
    * @returns Bedrock representation of the message.
```

Comment:
Not from this pr but we should move away from using jsdoc comments because typescript can handle this better